### DISCUSSION TITLE
Missing optional dependency 'sklearn.linear_model'.  Use pip or conda to install sklearn.linear_model.

### DISCUSSION THREAD

**Discussion by stormrageHF** [sinaptik-ai/pandas-ai#356] [Q&A]
But I already installed sklearn
Here is a portion of the generated code：

# Import libraries
import pandas as pd
from sklearn.linear_model import LinearRegression
from sklearn.model_selection import train_test_split

How do I solve this problem

**Comment by mspronesti** [comment]
Can you share the exact snippet your are using and the generated code ?
